tests/e2e/support/commands.js

```diff
@@ -77,7 +77,11 @@ Cypress.Commands.add('requestGameList', () => {
 Cypress.Commands.add('setupGameAsP0', (alreadyAuthenticated = false, isRanked = false) => {
   if (!alreadyAuthenticated) {
     cy.wipeDatabase();
-    cy.visit('/');
+    cy.visit('/', {
+      onBeforeLoad(win) {
+        delete win.navigator.__proto__.serviceWorker;
+      },
+    });
```

Comment:
There is an open issue for cypress hanging up on the service worker. It causes a timeout error on the cy.visit(). Currently this is the only way I can bypass that error 

https://github.com/cypress-io/cypress/issues/27501

Reply:
IMO this feels hacky and will be annoying to do in every `cy.visit()` inside the test